Makefile

```diff
@@ -90,10 +90,24 @@ all: image
 
 build:
 	@mkdir -p bin
-	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/...
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-gc
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-worker
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-master
+	$(GO_CMD) build -v -o bin $(BUILD_FLAGS) ./cmd/nfd-topology-updater
 
 install:
-	$(GO_CMD) install -v $(BUILD_FLAGS) ./cmd/...
+	$(GO_CMD) install -v $(BUILD_FLAGS) ./cmd/nfd-gc
+	$(GO_CMD) install -v $(BUILD_FLAGS) ./cmd/nfd-worker
+	$(GO_CMD) install -v $(BUILD_FLAGS) ./cmd/nfd-master
+	$(GO_CMD) install -v $(BUILD_FLAGS) ./cmd/nfd-topology-updater
+
+build-kubectl-plugin:
```

Comment:
How about rename this to more generic `build-tools` or smth?

Or alternatively we could make this a generic `build-%` target like

```
build-%:
    $(GO_CMD) build -v -o bin ./cmd/$*
```

Then you could also use (alternative to my previous comment):

```
BUILD_BINARIES := nfd-master nfd-worker nfd-topology-updater nfd-gc

build: $(foreach bin, $(BUILD_BINARIES), build-$(bin)):
```

Reply:
that's a better idea, yup.